cloto/tests/test_auth_manager.py

```diff
@@ -46,10 +67,14 @@ def setUp(self):
         self.url_noResponse = "http://127.0.0.1:35357/v2.0"
         self.url_server_error = "http://serverWithErrors.com:35357/v2.0"
         self.tenantId = "6571e3422ad84f7d828ce2f30373b3d4"
+        self.auth_api_v2 = "v2.0"
+        self.auth_api_v3 = "v3"
         self.a = AuthorizationManager.AuthorizationManager()
```

Comment:
Perfect candidates to be included in `constants.py` and be shared thorough the code.